crates/pallet-history-seeding/src/tests.rs

```diff
@@ -0,0 +1,100 @@
+use super::*;
+use crate::{self as pallet_history_seeding};
+use frame_support::{assert_noop, assert_ok, construct_runtime, derive_impl};
+use frame_system as system;
+use sp_runtime::BuildStorage;
+
+type Block = frame_system::mocking::MockBlock<Test>;
+
+construct_runtime!(
+    pub struct Test {
+        System: frame_system,
+        Sudo: pallet_sudo,
```

Comment:
I don't think we need sudo pallet here, the feature of limiting the account has no direct correlation with sudo account, let alone the pallet.